ansible/group_vars/nimbus.sepolia.yml

```diff
@@ -78,6 +81,13 @@ validator_client_build_frequency: 'daily'
 # Ports
 validator_client_metrics_port:    '{{ 8108 + idx|int + 1 }}'
 validator_client_keymanager_port: '{{ 5052 + idx|int + 1 }}'
+
+# Geth metrics¬
+geth_expo_service_name: '{{ geth_service_name }}'
+geth_expo_source_cont_name: '{{ geth_cont_name }}'
+geth_expo_source_data_path: '{{ geth_cont_vol }}/data'
+geth_expo_cont_port:  '{{ 9400 + (idx|int) + 1 }}'
```

Comment:
You're not opening the port on the firewall for this.

Reply:
fixed in latest commit